src/routes/backupTarget/BackupTargetList.js

```diff
@@ -0,0 +1,113 @@
+import { pagination } from '../../utils/page'
+
+function list({ loading, dataSource, deleteBackupTarget, editBackupTarget, rowSelection, height }) {
+  const columns = [
+    {
+      title: 'Name',
+      dataIndex: 'name',
+      key: 'name',
+      width: 150,
+      sorter: (a, b) => a.name.localeCompare(b.name),
+      render: (text) => {
+        return (
+          <div>{text}</div>
+        )
+      },
+    }, {
+      title: 'URL',
+      dataIndex: 'backupTargetURL',
+      key: 'backupTargetURL',
+      width: 250,
+      sorter: (a, b) => a.backupTargetURL.localeCompare(b.backupTargetURL),
+      render: (text) => {
+        return (
+          <div>{text}</div>
+        )
+      },
+    }, {
+      title: 'Credential Secret',
+      dataIndex: 'credentialSecret',
+      key: 'credentialSecret',
+      width: 150,
+      sorter: (a, b) => a.credentialSecret.localeCompare(b.credentialSecret),
+      render: (text) => {
+        return (
+          <div>{text}</div>
+        )
+      },
+    }, {
+      title: 'Poll Interval',
+      dataIndex: 'pollInterval',
+      key: 'pollInterval',
+      width: 100,
+      sorter: (a, b) => a.pollInterval.localeCompare(b.pollInterval),
+      render: (text) => {
+        return (
+          <div>{text}</div>
+        )
+      },
+    }, {
+      title: 'Available',
+      dataIndex: 'available',
+      key: 'available',
+      width: 80,
+      sorter: (a, b) => a.available - b.available,
+      render: (text) => {
+        return (
+          <div style={{ display: 'flex', justifyContent: 'center' }}>
+            {text === true ? (
+               <Tooltip title="This backup target is available to sync backup to it">
+                <Icon type="check-square" style={{ color: 'green', alignSelf: 'center' }} />
+              </Tooltip>
+            ) : (
+              <Tooltip title="This backup target is unavailable, please check the URL and credential secret are all correct.">
+                <Icon type="exclamation-circle" style={{ color: 'red', alignSelf: 'center' }} />
+              </Tooltip>
+            )}
+          </div>
+        )
+      },
+    }, {
+      title: 'Operation',
+      key: 'operation',
+      width: 100,
+      render: (text, record) => {
+        return (
+          <BackupTargetActions editBackupTarget={editBackupTarget} deleteBackupTarget={deleteBackupTarget} selected={record} />
+        )
+      },
+    },
+  ]
+
+  return (
+    <div id="backupTargetTable" style={{ flex: 1, height: '1px', overflow: 'hidden' }}>
+      <Table
+        className="common-table-class"
+        bordered={false}
+        columns={columns}
+        rowSelection={rowSelection}
+        dataSource={dataSource}
+        loading={loading}
+        simple
+        pagination={pagination('backupTargetPageSize')}
+        rowKey={record => record.id}
+        scroll={{ x: 970, y: dataSource.length > 0 ? height : 1 }}
+      />
+    </div>
```

Comment:
The scrollbar doesn't work well when the table header is taller on small screens.
![Screenshot 2024-12-17 at 6 46 13 PM (2)](https://github.com/user-attachments/assets/f6b31bc4-a7ed-4871-8331-4ce951fdca43)